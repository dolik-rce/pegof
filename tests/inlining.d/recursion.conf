input inlining.d/recursion.peg
optimize all
