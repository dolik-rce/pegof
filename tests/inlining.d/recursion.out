X <-
    A
    / B

A <- "x" A*

B <- "y" ("z" B*)*
